### PR TITLE
New landing page and favicon

### PR DESCRIPTION
**Pull request by Christiaanvdm**
Addresses ticket #43 

New solution implemented:
geocontextlanding

### CODE REVIEW

**Comment by timlinux** [comment]
Can you include a gif of the landing page experience please?

Thanks

File: django_project/base/templates/landing_page.html [hunk 1]
```diff
@@ -5,10 +5,75 @@
...
+    <div class="container">
+        <div class="jumbotron landing_header">
+            <h2 class="display-4 my-carousel-header" id="trigger-waypoint">Welcome to GeoContext!</h2>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row" style="margin-bottom: 5rem;">
+            <div class="col-sm-8">
+                <div class="jumbotron" id="trigger-diagram"><h3 class="text-center"> GeoContext is a service that will
+                    fetch interesting data
```
Comment (timlinux) [inline django_project/base/templates/landing_page.html]: Please clean up the formatting here

File: django_project/base/templates/landing_page.html [hunk 2]
```diff
@@ -5,10 +5,75 @@
...
+            <div class="col-sm"></div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row" style="margin-bottom: 5rem;">
+            <div class="col-sm"></div>
+            <div class="col-sm"></div>
+            <div class="col-sm-8">
+                <div class="jumbotron" id="trigger-terminal"><h3 class="text-center"> GeoContext can connect to various
+                    different types of
```
Comment (timlinux) [inline django_project/base/templates/landing_page.html]: and here...

**Review by timlinux — COMMENTED** [review]
Can you please clean up the formatting and remove any unused image resources.

**Review by timlinux — COMMENTED** [review]
LGTM

**Review by timlinux — APPROVED** [review]

**Comment by timlinux** [comment]
Great job thanks!

**Comment by anitanh** [comment]
hi @Christiaanvdm just wanted to give you a little tips when creating a PR :-)

at the top of your PR, if you write:
```
fix <issue link>
```
Your PR will be attached to the issue that you're working on in waffle like this for example:
screenshot from 2018-11-16 10-35-28

Also it is better to write what you actually did in the commit:
instead of: `Fix #geocontext-43 We need a more beautiful landing page`
maybe you could write: `change landing page layout` then the next commit maybe `add buttons` etc

**Comment by Christiaanvdm** [comment]
@ann26 

Roger. Thanks for the advice, will do that in that future 👍

**Pull request merged**